articles/custom-signup.md

```diff
@@ -49,92 +39,30 @@ Notice that `name` and `color` are custom fields.
 > Note that there is currently no way to validate user-supplied custom fields when signing up.
 Validation must be done from an Auth0 rule when logging in, or with custom logic in your application.
 
-## 2. Auth0.js and dependencies
-
-```html
-<script src="https://cdn.auth0.com/w2/auth0-6.7.js"></script>
-<script src="http://code.jquery.com/jquery-2.1.4.min.js"></script>
-```
-
-```js
-window.auth0 = new Auth0({
-  domain: '${account.namespace}',
-  clientID: '${account.clientId}'
-});
-```
-
-## 3. Submitting the form (popup mode)
-
-Here we are first signing up the user through Auth0. If the signup is successful,
-a call to `PATCH users/{user_id}` is made, which adds the custom fields to the
-`user_metadata` field of the user's profile.
-
-> This call to APIv2 is allowed from the client because all JWTs for logged-in
-users implicitly include the `update:current_user_metadata` scope by default.
-[This document describes how implicit scopes work when using APIv2](/api/v2/changes#scopes).
-
-```js
-$('#signup').submit(function (e) {
-  e.preventDefault();
-
-  function v2PatchUser (userId, id_token, data, successCallback, errorCallback) {
-    $.ajax({
-        method: 'patch',
-        url: apiEndpoint + 'users/' + userId,
-        dataType: 'json',
-        headers: {
-            'Authorization': 'Bearer ' + id_token
-        },
-        data: data,
-        success: successCallback,
-        error: errorCallback
-    });
-  }
-
-  function signupCallback (err, profile, id_token) {
-      if (err) {
-        alert('Something went wrong signing up: ' + err);
-      } else {
-        var data = {
-          user_metadata: {
-            favorite_color: $('#color').val(),
-            name: $('#name').val()
-          }
-        };
-        function updateSuccess () {
-          alert('Successfully signed up!');
-        }
-        function updateError (jqXHR) {
-          alert('Something went wrong signing up: ' + jqXHR.responseText);
-        }
-        v2PatchUser(profile.user_id, id_token, data, updateSuccess, updateError);
-      }
+### 2. Send the Form Data
+
+Send a POST request to the /dbconnections/signup endpoint in Auth0. You will need to send your `ClientId`, the `email` and `password` of the user being signed up and the custom fields as part of `user_metadata`.
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/dbconnections/signup",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{\"client_id\": \"${account.clientId}\",\"email\": \"$('#signup-email').val()\",\"password\": \"$('#signup-password').val()\",
+    \"user_metadata\": "{\"name\": \"john\",\"color\": \"red\"}"}"
```

Comment:
@angelaLazar hi, this JSON throws an error: `Error compiling document "custom-signup.md". Invalid JSON in HAR snippet.`